Source/Core/Duality/Drawing/Canvas.cs

```diff
@@ -394,6 +394,168 @@ public void DrawThickLine(float x, float y, float x2, float y2, float width)
 			this.DrawThickLine(x, y, 0, x2, y2, 0, width);
 		}
 
+		/// <summary>
+		/// Draws an image.
+		/// </summary>
+		public void DrawImage(ContentRef<Material> material, float x, float y, float z, float w, float h)
+		{
+			this.PushState();
+
+			if (w < 0.0f) { x += w; w = -w; }
+			if (h < 0.0f) { y += h; h = -h; }
+
+			Vector3 pos = new Vector3(x, y, z);
+
+			Vector2 shapeHandle = pos.Xy;
+			float offset = this.State.DepthOffset;
+			ColorRgba shapeColor = material.Res.MainColor;
```

Comment:
The material color shouldn't go into the vertex color, since material parameters [are applied in the shader](https://github.com/AdamsLair/duality/blob/b31fbab9c85bf3e018dd9025f510f0ae25a19784/Source/Core/Duality/EmbeddedResources/Minimal.vert#L17). Instead, use the `ColorTint` property of the canvas state, [as seen in other Canvas methods](https://github.com/AdamsLair/duality/blob/b31fbab9c85bf3e018dd9025f510f0ae25a19784/Source/Core/Duality/Drawing/Canvas.cs#L941).